global.json

```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "5.0.100-rc.1.20379.10"
+    "version": "5.0.100-rc.1.20429.2"
```

Comment:
`5.0.100-rc.1.20452.10` is the coming official SDK. Can we move to that instead❔ If not, do we plan to follow up w/ a shift to that version❔